src/Sulu/Component/Webspace/Loader/Exception/InvalidDefaultErrorTemplateException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
```

Comment:
yes i have copied from another file ... i think it is wrong in nearly all files!